.rhcicd/sonarqube/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Image available at https://images.paas.redhat.com/repository/alm/ubi9-java17-runtime.
+FROM images.paas.redhat.com/alm/ubi9-java17-runtime:mr39
```

Comment:
What is this image?

Reply:
It's a JRE17 image that is generated by the IT ALM team... It is the only image I found in the pass repo that had J17 that used ubi9...

Reply:
Do we need ubi9?

Reply:
Do we also have to rely on the pass repo?

Reply:
We could use a public image like https://catalog.redhat.com/software/containers/ubi9/openjdk-17-runtime/61ee7d45384a3eb331996bee